### ISSUE TITLE
frio: Improve Event Reminder display

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#4513]
Currently, the event reminders at the top of the network page can't be directly opened in a new tab, it just displays the modal content. Instead, they have to be clicked once to open the modal, then opened in a new tab using the link in the bottom right.

Additionally, I would like to hide the reminders from events I said I wouldn't be attending.

**Comment by rabuzarus** [comment]
What I would like to see for frios event reminder:

- A combination of birthday and events reminder, so that we would have only one reminder
- using the short date format of events which appear in the networkstream

**Comment by MrPetovan** [comment]
I’m not convinced birthdays should be treated as regular events as they are automatically generated.

For the date format, do you expect “in 2 days”, “in 6 hours”, “in 21 minutes”?

**Comment by rabuzarus** [comment]
Mhhh I thought it might be useful to have all the coming reminders in one view. So the user does not have to expand the one and then the other to get informed about important dates. And the gui would be less cluttered.

For frio event short date format I mean something like: Do 29 Feb 6:00 PM

**Comment by MrPetovan** [comment]
Here's what I'm expecting for birthday reminders with 550 contacts: a gallery of avatar pictures for the people whose birthday is exactly today on the side column. This way the birthday events don't linger for a week in the network page header.

**Comment by tobiasd** [comment]
That sounds like a widget/additional feature that should not be frio specific @MrPetovan

**Comment by MrPetovan** [comment]
Indeed, I should probably remove the `frio` tag since what I've already done in #4515 affects all the themes.

**Comment by rabuzarus** [comment]
yes it isn't specific to frio.

> That sounds like a widget

I have often thought about this. The best thing would be a widget. BUT a widget would break the logic in the network module. Here all widgets are some kind of filter widgets. So adding an event widget - which is more an informative widget - looks alien.

For the profile module a widget would be the best - especially if the profile is an organization account and the organization could show its public events to others
